src/instructlab/defaults.py

```diff
@@ -93,6 +93,10 @@ class _InstructlabDefaults:
     MISTRAL_GGUF_MODEL_NAME = "mistral-7b-instruct-v0.2.Q4_K_M.gguf"
     MODEL_REPO = "instructlab/granite-7b-lab"
     JUDGE_MODEL_MT = "prometheus-eval/prometheus-8x7b-v2.0"
+    JUDGE_MODEL_DK = "gpt-4o"
+    # Taken from DEFAULT_SYSTEM_PROMPT in eval repo
+    MODEL_EVAL_SYSTEM_PROMPT = "You are an advanced AI assistant designed to provide precise and accurate information. Your primary goal is to answer queries with the most up-to-date and factual information available. Focus on delivering clear, concise, and correct responses. If you're uncertain about any aspect of the query, state your level of confidence and provide the most accurate information you can. Your responses should prioritize accuracy over all other considerations."
```

Comment:
i think this should be moved to https://github.com/instructlab/instructlab/blob/main/src/instructlab/common.py along with the other sys prompts

Reply:
@jaideepr97 That's a very good point. @alimaredia we should definitely use the default templates if possible.

Reply:
@RobotSail If you think we should use a different prompt, which one should we use from here: https://github.com/instructlab/instructlab/blob/main/src/instructlab/common.py

My original thought was that since this was what you added in the eval library that should be the default for what we use in this evaluation.

Is this something we can revisit in future work? Ideally based on the model provided to `--model` we should probably choose the prompt automatically?

@jaideepr97 I'd be happy to move this constant into common.py for now.

Reply:
@alimaredia You don't need to select it yourself, you can just use this logic to figure that out: https://github.com/instructlab/instructlab/blob/3565ba962adc22576716408c9d1941b17b39abe7/src/instructlab/model/evaluate.py#L308

Reply:
Great, then in that case I'll set the default for the flag & config var to `None` and if the value of that is `None` I'll pass in the value from this system prompt selection function.

Reply:
@alimaredia I love that approach.